requirements.txt

```diff
@@ -3,7 +3,7 @@ coveralls==0.5
 Django==1.7.4
 docopt==0.6.2
 flake8==3.4.1
-psycopg2==2.5.4
+psycopg2==2.7.3
```

Comment:
did you test w/ fresh setup after this?

Reply:
Yes, it states that django secret key is to be used in place of secret key.